Assets/Source/Components/Environment/ParallaxComponent.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Assets.Source.Components.Base;
+using Assets.Source.Constants;
+using UnityEngine;
+
+namespace Assets.Source.Components.Environment
+{
+    public class ParallaxComponent : ComponentBase
+    {
+        private readonly int LAYERS_PER_GROUP = 3; //just in case we decide to change it eventually
+        private readonly int SORTING_ORDER = -10; //we want these images to be behind everything else
+
+        [System.Serializable]
+        public struct ParallaxInfo
+        {
+            public string name;
+            public Sprite sprite;
+            [Range(1, 1000)]
+            [Tooltip("1 will not move at all, 1000 will move the fastest. Calculates speed using log of this value")]
+            public float parallaxEffect;
+        };
+
+        [Tooltip("Each Parallax Info is the information needed to generate one layer of the parallax effect")]
+        public ParallaxInfo[] parallaxInfos;
```

Comment:
this is still a public field tho.  If its needed externally, encapsulate it in a property.  If not, mark it private and use [SerializeField]